src/index.ts

```diff
@@ -1070,7 +1071,7 @@ class Workflow {
         content_type: `application/vnd.relay.sdk.info+json`,
         analytics_content: {
```

Comment:
I think this field needs to be named 'content', per https://bitbucket.org/relaypro/relay_erlang/src/master/priv/wf_api_log_analytics_event_request.json

Reply:
One more finding, as I'm trying to run this: the "content" value needs to be a string, so it should be wrapped in JSON.stringify:

  private _logSdkInfo() {
    setTimeout(() => {
      this._cast(`log_analytics_event`, {
        category: `sdk-info`,
        content_type: `application/vnd.relay.sdk.info+json`,
        content: JSON.stringify({
          language: `relay-js`,
          version: version,
        })
      })
    }, 0)
  }